FirebaseVertexAI/Sources/GenerativeModel.swift

```diff
@@ -252,7 +252,7 @@ public final class GenerativeModel {
   /// [zero-shot](https://developers.google.com/machine-learning/glossary/generative#zero-shot-prompting)
   /// or "direct" prompts. For
   /// [few-shot](https://developers.google.com/machine-learning/glossary/generative#few-shot-prompting)
-  /// input, see ``countTokens(_:)-9spwl``.
+  /// input, see ``countTokens(_:)``.
```

Comment:
nit: `countTokens(_ content: @autoclosure () throws -> [ModelContent])` and single backticks.

Reply:
Done.